### PR TITLE
Add docs on how to move data from an NFS server to AzureFile NFS storage

### PR DESCRIPTION
**Pull request by GeorgianaElena**
Docs for https://github.com/2i2c-org/infrastructure/issues/876

### CODE REVIEW

File: docs/howto/operate/move-hub.md
Comment (damianavila) [inline docs/howto/operate/move-hub.md:116]: This note just at the beginning of the Azure Files section feels a little bit weird... I mean, it seems the result of some previous paragraph discussing something and that paragraph does not exist above, AFAIK.
I am missing something, maybe?

File: docs/howto/operate/move-hub.md
Comment (sgibson91) [inline docs/howto/operate/move-hub.md:61]:
```suggestion
For long-running tasks, especially those running on a remote machine, we recommend using [`screen`](https://www.gnu.org/software/screen/manual/screen.html). Screen is a utility that allows starting `screen` sessions that can then be put to run in the background with the possibility of re-attaching to them. Processes running in `screen` will continue to run in the background even if you get disconnected.
```

File: docs/howto/operate/move-hub.md
Comment (sgibson91) [reply docs/howto/operate/move-hub.md:116]: Yeah, I'd maybe rephrase it as "You may be tempted to attach an existing NFS server across two clusters. However, this is not possible since Azure VMs cannot simultaneously exist in two virtual networks and each cluster will have it's own network."

File: docs/howto/operate/move-hub.md
Comment (sgibson91) [inline docs/howto/operate/move-hub.md:119]:
```suggestion
We also use AzureFiles as in-cluster NFS storage. These are the steps to transfer the home directories from a NFS server located in another cluster, but in the same Azure subscription, into AzureFile NFS storage.
```

File: docs/howto/operate/move-hub.md
Comment (sgibson91) [inline docs/howto/operate/move-hub.md:126]: General note that shouldn't block this PR.

We should update our Azure cluster creation docs to include a step that creates a new SSH key pair and uploads the sops-encrypted, private part to the `secrets` directory. Atm, I think Georgiana and I are using our own ssh keys which is unsustainable.

File: docs/howto/operate/move-hub.md
Comment (sgibson91) [inline docs/howto/operate/move-hub.md:130]:
```suggestion
   2. Make sure `kubectl` is authenticated with the Azure cluster that hosts the source VM, as we'll be 'hopping' through it to access the NFS VM.
```

File: docs/howto/operate/move-hub.md
Comment (sgibson91) [inline docs/howto/operate/move-hub.md:132]: You may wish to add a note here that suggests using `kubectl config get-contexts` to list which contexts are available. Also, if the desired context isn't available, you can authenticate using `az aks get-credentials --name CLUSTER_NAME --resource-group RESOURCE_GROUP_NAME`

File: docs/howto/operate/move-hub.md
Comment (sgibson91) [inline docs/howto/operate/move-hub.md:134]:
```suggestion
   3. Ssh into the source NFS VM using the [`./terraform/proxycommand.py`](https://github.com/2i2c-org/infrastructure/blob/master/terraform/azure/proxycommand.py) script, passing it the private key from step 1, the authorized user to connect to the VM using this key pair (this is usually `hubadmin` or `hub-admin`, and can be found in the terraform config `azurerm_kubernetes_cluster.linux_profile.admin_username`), and the address of the NFS VM.
```

Can we turn the `azurerm_***` part into a link please?

**Review by sgibson91 — APPROVED** [review]
Thanks @GeorgianaElena, this is really great! I just have a few comments :)

File: docs/howto/operate/move-hub.md
Comment (GeorgianaElena) [reply docs/howto/operate/move-hub.md:116]: Great rephrasing @sgibson91, thanks! I will use it, if that's ok with you :D 

> This note just at the beginning of the Azure Files section feels a little bit weird... I mean, it seems the result of some previous paragraph discussing something and that paragraph does not exist above, AFAIK.
I am missing something, maybe?

@damianavila, I was refering to the paragraph [here](https://github.com/2i2c-org/infrastructure/blob/master/docs/howto/operate/move-hub.md#copy-home-directories):
*"This might not entirely be necessary - if the source and target cluster
are in the same GCP Project / AWS Account, we can just re-use the same
home directory storage!"*

But now I see I didn't do a good job capturing that idea :sweat_smile: I will go with Sarah's rephrasing, since that's much better.

Thank you! :sparkles:

File: docs/howto/operate/move-hub.md
Comment (GeorgianaElena) [reply docs/howto/operate/move-hub.md:126]: > Atm, I think Georgiana and I are using our own ssh keys which is unsustainable.

Yessss! I guess we could still upload them to the `secrets` dir and generate new personal ones to fix this mhm... (But that kind of has consequences :grimacing: )

*I think* that if we change them now in terraform with a it will force a resource recreation (as per terraform's docs) of the cluster.

Absolutely  + 1 on documenting this!!!

Comment (sgibson91) [reply docs/howto/operate/move-hub.md:126]: Tbh, right now there's more than just the SSH keys blocking other team members from working on the UToronto and CarbonPlan clusters 😅 so I wouldn't worry too much about those specific clusters now and instead focus on not generating future toil

**Comment by GeorgianaElena** [comment]
Thanks @damianavila and @sgibson91 for reviewing this! I hope I've addressed everything!
Since I have one approval, I will merge this now :rocket:

File: docs/howto/operate/move-hub.md
Comment (GeorgianaElena) [reply docs/howto/operate/move-hub.md:126]: Done! Thank you! :sparkles:

**Pull request merged**
